### ISSUE TITLE
Strongbox downgrade?

### ISSUE THREAD

**Issue by Ben-Voris** [strongbox-password-safe/Strongbox#529]
Each time I start strongbox, I get this message:
![Strongbox Downgrade](https://user-images.githubusercontent.com/31499002/122258881-ccb4a200-ce8e-11eb-9d0e-fa4c3600edb0.png)￼

I click on Upgrade, restore my purchase, and I can go on. But, the next time I start strongbox it appears again.

```
￼ Strongbox 1.52.21 Debug Information at 2021-06-16 10:38:51.870
--------------------
App Version: com.markmcguill.strongbox [1.52.21 (3081)@7ca2a5b]
--------------------
Device
--------------------
Model: iPhone
CPU: ARM64E
System Name: iOS
System Version: 14.6
--------------------
Preferences
--------------------
autoFillDidChangeDatabases: 0
showDatabaseIcon: 1
PKKeychainVersionKey: 4
AKLastCheckInAttemptDate: 2018-10-17 02:21:08 +0000
iCloudPrompted: 1
haveAskedAboutBackupSettings: 1
PKUserHasDisabledPeerPaymentKey: 1
allowThirdPartyKeyboards: 0
mainAppDidChangeDatabases: 1
disallowAllPinCodeOpens: 1
userHasOptedInToThirdPartyStorageLibraries: 1
backupFiles: 1
AKLastCheckInSuccessDate: 2018-10-17 02:21:08 +0000
migratedToNewPasswordGenerator: 1
tryDownloadFavIconForNewRecord: 1
migratedLocalDatabasesToNewSystem: 1
hasShownAutoFillLaunchWelcome: 1
migratedToNewQuickLaunchSystem: 1
hideExportFromDatabaseContextMenu: 0
autoFillLastKnownGoodDatabaseState: {length = 32, bytes = 0x175c3e4e af665364 171e96ad 8ffeb13b ... 8d312f72 1aa886bd }
useQuickLaunchAsRootView: 0
scheduledTipsCheckDone: 1
installDate: 2018-07-23 22:21:40 +0000
hasShownFirstRunWelcome: 1
kMigratedYubiKeyEmergencyWorkaroundsToVirtualKeys: 1
GMSInstanceID-version: 1.0.10
autoLockTimeSeconds: 600
WebKitShrinksStandaloneImagesToFit: 1
promptedForCopyPasswordGesture: 1
appLockAllowDevicePasscodeFallbackForBio: 1
lastKnownGoodDatabaseState: {length = 32, bytes = 0x32e654b9 f17362e1 0663c6f9 1d02c1a9 ... 82e63b1b 6ad7a8e3 }
autoFillQuickLaunchUuid: CCA99D80-90AA-4160-A936-96A7563D05C1
INNextHearbeatDate: 561652125.547388
PKHandoffPaymentsDisabled: 1
AKLastEmailListRequestDateKey: 2018-09-20 00:12:36 +0000
databaseCellTopSubtitle: 3
legacyShowMetadataOnDetailsScreen: 1
showDatabasesSeparator: 1
lastEntitlementCheckAttempt: 2021-06-16 16:30:04 +0000
AKLastIDMSEnvironment: 0
showPasswordByDefaultOnEditScreen: 1
fullFileProtection: 1
iCloudOn: 1
migratedToNewSecretStore: 1
hideTotp: 0
appLockDelay2.0: 60
easyReadFontForAll: 1
applicationIdentifierPrefix: 4326J8XDF2
safesList: {length = 8617, bytes = 0x62706c69 73743030 d4000100 02000300 ... 00000000 00001d77 }
autoFillExitedCleanly: 1
hasMigratedDatabaseSubtitlesAug2020: 1
hideEmptyFieldsInDetailsView: 1
numberOfEntitlementCheckFails: 0
showAllFilesInLocalKeyFiles: 0
launchCount: 635
AKDeviceUnlockState: 1
collapsedSections: (
    0,
    0,
    0,
    0,
    0,
    0
)
syncPullEvenIfModifiedDateSame: 0
neverShowForMacAppMessage: 1
isPro: 1
AddingEmojiKeybordHandled: 1
MSVLoggingMasterSwitchEnabledKey: 0
colorizeUseColorBlindPalette: 0
PKContactlessInterfaceHomeButtonSourceHasOccuredKey: 1
endFreeTrialDate: 2017-09-25 17:14:00 +0000
appLockMode2.0: 2
copyPasswordOnLongPress: 1
migratedQuickLaunchToAutoFill: 1
isHavePromptedAboutFreeTrial: 1
clearClipboardEnabled: 1
WebKitOfflineWebApplicationCacheEnabled: 1
safes: (
        {
        fileIdentifier = 0BwcD0j0uWrgDeW1LV3VKTmNTOVk;
        fileName = "BenPasswords.psafe3";
        isEnrolledForTouchId = 1;
        isTouchIdEnabled = 1;
        nickName = Ben;
        offlineCacheAvailable = 1;
        offlineCacheEnabled = 1;
        offlineCacheFileIdentifier = "759A00C1-B63C-4C9C-9849-A86115AA2BD0";
        storageProvider = 0;
    },
        {
        fileIdentifier = 0BwcD0j0uWrgDeW1LV3VKTmNTOVk;
        fileName = "LesliePasswords.psafe3";
        isEnrolledForTouchId = 1;
        isTouchIdEnabled = 1;
        nickName = Leslie2;
        offlineCacheAvailable = 1;
        offlineCacheEnabled = 1;
        offlineCacheFileIdentifier = "2B5A4A9C-5D7D-489F-AAC7-E9937CDDBDB6";
        storageProvider = 0;
    },
        {
        fileIdentifier = 0AAcD0j0uWrgDUk9PVA;
        fileName = "pwsafe.psafe3";
        isEnrolledForTouchId = 1;
        isTouchIdEnabled = 1;
        nickName = "Dwight's";
        offlineCacheAvailable = 1;
        offlineCacheEnabled = 1;
        offlineCacheFileIdentifier = "733AD06F-6CCC-42B5-9D0A-43C5EA1E6533";
        storageProvider = 0;
    }
)
hideTips: 1
GID_AppHasRunBefore: 1
DidCleanUpOldOfflineCacheFiles: 1
databaseCellSubtitle2: 1
showDatabaseStatusIcon: 1
failedUnlockAttempts: 0
hasDoneProFamilyCheck: 1
backupIncludeImportedKeyFiles: 0
clipboardHandoff: 0
iCloudWasOn: 1
KeychainAccessibilityMigration: 1
instantPinUnlocking: 1
appLockAppliesToPreferences: 0
WebDatabaseDirectory: /var/mobile/Containers/Data/Application/44050663-2140-4C8E-B9D4-01ABCE3823B7/Library/Caches
monitorInternetConnectivity: 1
promptedForReview: 1
WebKitLocalStorageDatabasePathPreferenceKey: /var/mobile/Containers/Data/Application/44050663-2140-4C8E-B9D4-01ABCE3823B7/Library/Caches
migratedOfflineDetectedBehaviour: 1
safesMigratedToNewSystem: 1
DidMigrateToAppGroups: 1
clearClipboardAfterSeconds: 60
haveAttemptedMigrationToFullFileProtection: 1
INNextFreshmintRefreshDateKey: 561440901.526489
hideKeyFileOnUnlock: 0
syncForcePushDoNotCheckForConflicts: 0
migratedV1Dropbox: 1
migratedDatabasesToNewStore: 1
Ep: 1532384500
Flags: P[1[-1358]110[635]1111]
--------------------
Last Crash
--------------------
--------------------
Sync
--------------------
Ben (Google Drive Sync) => 2021-06-15 20:15:05.943 (62 KB)
Leslie (Google Drive Sync) => 2021-05-31 09:08:04.593 (66 KB)
Dwight's (Google Drive Sync) => 2019-01-03 12:54:24.382 (6 KB)
Ben-HP (OneDrive Sync) => 2021-06-10 10:59:12.000 (10 KB)
Copy of BenPasswords (Local Sync) => 2021-02-01 14:25:46.680 (61 KB)
--------------------
[Support/F463413E-2DA8-4D09-9233-7CC7766B3B2C-offline-cache.dat] 7 KB - 6/18/19, 11:48:45
[Support/B2AE174D-0ABC-4F7D-B558-4DFF63D3264D-offline-cache.dat] 6 KB - 10/25/19, 14:34:30
[Support/B2CE22F8-59B1-4C10-B54B-4D0CDE68D60F-offline-cache.dat] 57 KB - 7/1/20, 10:04:11
[Support/1498F930-CFEC-4FC5-89B2-CA3771D75503-offline-cache.dat] 7 KB - 6/18/19, 10:43:26
[Support/CCA99D80-90AA-4160-A936-96A7563D05C1-offline-cache.dat] 56 KB - 7/2/20, 11:51:51
[Support/9F29D37F-D432-4B2D-AAC6-ABCF4610E175-offline-cache.dat] 8 KB - 3/23/19, 21:51:30
[Support/FB01B851-70A0-4BC2-BB12-62B7F01D525D-offline-cache.dat] 7 KB - 6/16/20, 14:10:23
[preferences/databases.json] 18 KB - Today, 10:28:52
[.com.apple.mobile_container_manager.metadata.plist] 538 bytes - 12/3/20, 08:58:15
[auto-fill-caches/B2AE174D-0ABC-4F7D-B558-4DFF63D3264D-autofill-cache.dat] 6 KB - 10/25/19, 14:34:30
[auto-fill-caches/4400026B-8829-430E-B668-EBBE175019F5-autofill-cache.dat] 648 bytes - 5/27/20, 09:52:32
[auto-fill-caches/FC44FAC2-AE43-4488-97EA-AB81A0AD37DE-autofill-cache.dat] 49 KB - 11/16/18, 17:01:27
[auto-fill-caches/9F29D37F-D432-4B2D-AAC6-ABCF4610E175-autofill-cache.dat] 8 KB - 3/23/19, 21:51:30
[auto-fill-caches/FB01B851-70A0-4BC2-BB12-62B7F01D525D-autofill-cache.dat] 7 KB - 6/16/20, 14:10:23
[auto-fill-caches/B2CE22F8-59B1-4C10-B54B-4D0CDE68D60F-autofill-cache.dat] 57 KB - 7/1/20, 10:04:11
[auto-fill-caches/1498F930-CFEC-4FC5-89B2-CA3771D75503-autofill-cache.dat] 7 KB - 6/18/19, 11:59:10
[auto-fill-caches/CCA99D80-90AA-4160-A936-96A7563D05C1-autofill-cache.dat] 56 KB - 7/2/20, 11:51:51
[auto-fill-caches/F463413E-2DA8-4D09-9233-7CC7766B3B2C-autofill-cache.dat] 7 KB - 6/18/19, 11:48:45
[Library/Preferences/group.strongbox.mcguill.plist] 16 KB - Today, 10:38:41
[Copy of BenPasswords.psafe3] 61 KB - 2/1/21, 14:25:46
[backups/51FE0E2D-E9C2-4FBA-9749-304EFA4DFF0B/2021-02-01T14:25:46-07:00.bak] 60 KB - 2/1/21, 14:25:06
[backups/1081AE9A-7E68-42F6-9EA1-331E5F42766A/2021-06-10T10:58:44-06:00.bak] 9 KB - 6/6/21, 20:52:48
[backups/1081AE9A-7E68-42F6-9EA1-331E5F42766A/2021-06-06T20:59:17-06:00.bak] 9 KB - 5/17/21, 09:34:20
[backups/1081AE9A-7E68-42F6-9EA1-331E5F42766A/2021-02-01T10:04:03-07:00.bak] 8 KB - 10/13/20, 20:01:51
[backups/1081AE9A-7E68-42F6-9EA1-331E5F42766A/2021-05-17T11:37:45-06:00.bak] 9 KB - 2/16/21, 16:26:34
[backups/1081AE9A-7E68-42F6-9EA1-331E5F42766A/2021-06-10T10:59:10-06:00.bak] 10 KB - 6/10/21, 10:58:47
[backups/1081AE9A-7E68-42F6-9EA1-331E5F42766A/2021-02-01T15:59:45-07:00.bak] 8 KB - 12/15/20, 09:25:32
[backups/1081AE9A-7E68-42F6-9EA1-331E5F42766A/2021-02-16T16:26:31-07:00.bak] 8 KB - 2/1/21, 15:58:49
[backups/B2CE22F8-59B1-4C10-B54B-4D0CDE68D60F/2021-05-22T11:21:59-06:00.bak] 65 KB - 4/18/21, 09:01:10
[backups/B2CE22F8-59B1-4C10-B54B-4D0CDE68D60F/2021-02-23T17:26:01-07:00.bak] 62 KB - 2/9/21, 18:39:09
[backups/B2CE22F8-59B1-4C10-B54B-4D0CDE68D60F/2021-04-01T17:29:07-06:00.bak] 62 KB - 2/21/21, 23:20:10
[backups/B2CE22F8-59B1-4C10-B54B-4D0CDE68D60F/2021-06-07T10:12:51-06:00.bak] 65 KB - 5/17/21, 17:14:00
[backups/B2CE22F8-59B1-4C10-B54B-4D0CDE68D60F/2021-04-23T15:13:15-06:00.bak] 64 KB - 3/31/21, 13:53:36
[backups/CCA99D80-90AA-4160-A936-96A7563D05C1/2021-06-06T21:16:13-06:00.bak] 61 KB - 5/17/21, 17:25:00
[backups/CCA99D80-90AA-4160-A936-96A7563D05C1/2021-03-29T23:18:54-06:00.bak] 61 KB - 3/12/21, 07:03:09
[backups/CCA99D80-90AA-4160-A936-96A7563D05C1/2021-06-06T21:20:22-06:00.bak] 61 KB - 5/31/21, 14:08:14
[backups/CCA99D80-90AA-4160-A936-96A7563D05C1/2021-06-16T07:35:32-06:00.bak] 61 KB - 6/6/21, 21:20:25
[backups/CCA99D80-90AA-4160-A936-96A7563D05C1/2021-03-31T09:15:16-06:00.bak] 61 KB - 3/19/21, 14:01:27
[sync-manager/local/B2AE174D-0ABC-4F7D-B558-4DFF63D3264D] 6 KB - 1/3/19, 12:54:24
[sync-manager/local/51FE0E2D-E9C2-4FBA-9749-304EFA4DFF0B] 61 KB - 2/1/21, 14:25:46
[sync-manager/local/1081AE9A-7E68-42F6-9EA1-331E5F42766A] 10 KB - 6/10/21, 10:59:12
[sync-manager/local/B2CE22F8-59B1-4C10-B54B-4D0CDE68D60F] 66 KB - 5/31/21, 09:08:04
[sync-manager/local/CCA99D80-90AA-4160-A936-96A7563D05C1] 62 KB - Yesterday, 20:15:05
--------------------
[Ben] on [Google Drive] Config
{
    auditConfig =     {
        auditInBackground = 1;
        caseInsensitiveMatchForDuplicates = 1;
        checkForCommonPasswords = 1;
        checkForDuplicatedPasswords = 1;
        checkForLowEntropy = 0;
        checkForMinimumLength = 0;
        checkForNoPasswords = 1;
        checkForSimilarPasswords = 1;
        checkHibp = 0;
        hibpCaveatAccepted = 0;
        hibpCheckForNewBreachesIntervalSeconds = 86400;
        lastKnownAuditIssueCount = 7;
        levenshteinSimilarityThreshold = "0.75";
        lowEntropyThreshold = 36;
        minimumLength = 12;
        showAuditPopupNotifications = 1;
        showCachedHibpHits = 1;
    };
    autoFillConvenienceAutoUnlockTimeout = 180;
    autoFillCopyTotp = 1;
    autoFillEnabled = 1;
    autoFillLastUnlockedAt = "645543279.735659";
    autoFillOnboardingDone = 1;
    autoLockOnDeviceLock = 1;
    autoLockTimeoutSeconds = 300;
    browseItemSubtitleField = 1;
    browseSortField = 1;
    browseSortFoldersSeparately = 1;
    browseSortOrderDescending = 0;
    browseViewType = 0;
    colorizePasswords = 1;
    colorizeProtectedCustomFields = 0;
    conflictResolutionStrategy = 0;
    convenienceExpiryOnboardingDone = 1;
    convenienceExpiryPeriod = 336;
    couldNotConnectBehaviour = 0;
    detailsViewCollapsedSections =     (
        0,
        0,
        0,
        0,
        0,
        0
    );
    doNotShowRecycleBinInBrowse = 0;
    doubleTapAction = 3;
    duressAction = 0;
    easyReadFontForAll = 1;
    emptyOrNilPwPreferNilCheckFirst = 0;
    failedPinAttempts = 0;
    fileIdentifier = 0BwcD0j0uWrgDeW1LV3VKTmNTOVk;
    fileName = "BenPasswords.psafe3";
    forceOpenOffline = 0;
    hasAcknowledgedAppLockBiometricQuickLaunchCoalesceIssue = 0;
    hasBeenPromptedForConvenience = 1;
    hasBeenPromptedForQuickLaunch = 1;
    hasShownInitialOnboardingScreen = 1;
    hasUnresolvedConflicts = 0;
    hideIconInBrowse = 0;
    hideTotp = 0;
    hideTotpCustomFieldsInViewMode = 1;
    hideTotpInBrowse = 0;
    immediateSearchOnBrowse = 0;
    isEnrolledForConvenience = 1;
    isTouchIdEnabled = 1;
    keePassIconSet = 0;
    keyFileBookmark = "<Not Set>";
    lastSyncAttempt = "645543329.19916";
    lastSyncRemoteModDate = "645502505.943";
    likelyFormat = 0;
    lockEvenIfEditing = 0;
    longPressTapAction = 4;
    makeBackups = 1;
    maxBackupKeepCount = 5;
    nextScheduledExport = "645984682.445853";
    nickName = Ben;
    offlineDetectedBehaviour = 0;
    onboardingDoneHasBeenShown = 1;
    promptedForAutoFetchFavIcon = 0;
    quickTypeDisplayFormat = 0;
    quickTypeEnabled = 1;
    readOnly = 0;
    scheduleExportIntervalDays = 14;
    scheduledExport = 1;
    scheduledExportOnboardingDone = 1;
    showChildCountOnFolderInBrowse = 0;
    showConvenienceExpiryMessage = 1;
    showEmptyFieldsInDetailsView = 1;
    showExpiredInBrowse = 1;
    showExpiredInSearch = 1;
    showFlagsInBrowse = 1;
    showKeePass1BackupGroup = 0;
    showPasswordByDefaultOnEditScreen = 1;
    showQuickViewExpired = 0;
    showQuickViewFavourites = 1;
    showQuickViewNearlyExpired = 1;
    showRecycleBinInSearchResults = 0;
    storageProvider = 0;
    tapAction = 1;
    tripleTapAction = 8;
    tryDownloadFavIconForNewRecord = 1;
    uuid = "CCA99D80-90AA-4160-A936-96A7563D05C1";
}
[Leslie] on [Google Drive] Config
{
    auditConfig =     {
        auditInBackground = 1;
        caseInsensitiveMatchForDuplicates = 1;
        checkForCommonPasswords = 1;
        checkForDuplicatedPasswords = 1;
        checkForLowEntropy = 0;
        checkForMinimumLength = 0;
        checkForNoPasswords = 1;
        checkForSimilarPasswords = 1;
        checkHibp = 0;
        hibpCaveatAccepted = 0;
        hibpCheckForNewBreachesIntervalSeconds = 86400;
        lastKnownAuditIssueCount = 80;
        levenshteinSimilarityThreshold = "0.75";
        lowEntropyThreshold = 36;
        minimumLength = 12;
        showAuditPopupNotifications = 1;
        showCachedHibpHits = 1;
    };
    autoFillConvenienceAutoUnlockTimeout = 180;
    autoFillCopyTotp = 1;
    autoFillEnabled = 1;
    autoFillLastUnlockedAt = "645553116.805699";
    autoFillOnboardingDone = 1;
    autoLockOnDeviceLock = 1;
    autoLockTimeoutSeconds = 600;
    browseItemSubtitleField = 1;
    browseSortField = 1;
    browseSortFoldersSeparately = 1;
    browseSortOrderDescending = 0;
    browseViewType = 0;
    colorizePasswords = 1;
    colorizeProtectedCustomFields = 0;
    conflictResolutionStrategy = 0;
    convenienceExpiryOnboardingDone = 1;
    convenienceExpiryPeriod = "-1";
    couldNotConnectBehaviour = 0;
    detailsViewCollapsedSections =     (
        0,
        0,
        0,
        0,
        0,
        0
    );
    doNotShowRecycleBinInBrowse = 0;
    doubleTapAction = 3;
    duressAction = 0;
    easyReadFontForAll = 1;
    emptyOrNilPwPreferNilCheckFirst = 0;
    failedPinAttempts = 0;
    fileIdentifier = 0BwcD0j0uWrgDeW1LV3VKTmNTOVk;
    fileName = "LesliePasswords.psafe3";
    forceOpenOffline = 0;
    hasAcknowledgedAppLockBiometricQuickLaunchCoalesceIssue = 0;
    hasBeenPromptedForConvenience = 1;
    hasBeenPromptedForQuickLaunch = 1;
    hasShownInitialOnboardingScreen = 1;
    hasUnresolvedConflicts = 0;
    hideIconInBrowse = 0;
    hideTotp = 0;
    hideTotpCustomFieldsInViewMode = 1;
    hideTotpInBrowse = 0;
    immediateSearchOnBrowse = 0;
    isEnrolledForConvenience = 1;
    isTouchIdEnabled = 1;
    keePassIconSet = 0;
    keyFileBookmark = "<Not Set>";
    lastSyncAttempt = "644775168.478465";
    lastSyncRemoteModDate = "644166484.5930001";
    likelyFormat = 0;
    lockEvenIfEditing = 0;
    longPressTapAction = 4;
    makeBackups = 1;
    maxBackupKeepCount = 5;
    nextScheduledExport = "647194379.206141";
    nickName = Leslie;
    offlineDetectedBehaviour = 0;
    onboardingDoneHasBeenShown = 1;
    promptedForAutoFetchFavIcon = 0;
    quickTypeDisplayFormat = 0;
    quickTypeEnabled = 1;
    readOnly = 1;
    scheduleExportIntervalDays = 28;
    scheduledExport = 1;
    scheduledExportOnboardingDone = 1;
    showChildCountOnFolderInBrowse = 1;
    showConvenienceExpiryMessage = 1;
    showEmptyFieldsInDetailsView = 0;
    showExpiredInBrowse = 1;
    showExpiredInSearch = 1;
    showFlagsInBrowse = 1;
    showKeePass1BackupGroup = 0;
    showPasswordByDefaultOnEditScreen = 1;
    showQuickViewExpired = 0;
    showQuickViewFavourites = 1;
    showQuickViewNearlyExpired = 1;
    showRecycleBinInSearchResults = 0;
    storageProvider = 0;
    tapAction = 1;
    tripleTapAction = 8;
    tryDownloadFavIconForNewRecord = 1;
    uuid = "B2CE22F8-59B1-4C10-B54B-4D0CDE68D60F";
}
[Dwight's] on [Google Drive] Config
{
    auditConfig =     {
        auditInBackground = 1;
        caseInsensitiveMatchForDuplicates = 1;
        checkForCommonPasswords = 1;
        checkForDuplicatedPasswords = 1;
        checkForLowEntropy = 0;
        checkForMinimumLength = 0;
        checkForNoPasswords = 1;
        checkForSimilarPasswords = 1;
        checkHibp = 0;
        hibpCaveatAccepted = 0;
        hibpCheckForNewBreachesIntervalSeconds = 86400;
        lastKnownAuditIssueCount = 2;
        levenshteinSimilarityThreshold = "0.75";
        lowEntropyThreshold = 36;
        minimumLength = 12;
        showAuditPopupNotifications = 1;
        showCachedHibpHits = 1;
    };
    autoFillConvenienceAutoUnlockTimeout = "-1";
    autoFillCopyTotp = 1;
    autoFillEnabled = 1;
    autoFillOnboardingDone = 1;
    autoLockOnDeviceLock = 1;
    autoLockTimeoutSeconds = 600;
    browseItemSubtitleField = 1;
    browseSortField = 1;
    browseSortFoldersSeparately = 1;
    browseSortOrderDescending = 0;
    browseViewType = 0;
    colorizePasswords = 1;
    colorizeProtectedCustomFields = 0;
    conflictResolutionStrategy = 0;
    convenienceExpiryOnboardingDone = 0;
    convenienceExpiryPeriod = "-1";
    couldNotConnectBehaviour = 0;
    detailsViewCollapsedSections =     (
        0,
        0,
        0,
        0,
        0,
        0
    );
    doNotShowRecycleBinInBrowse = 0;
    doubleTapAction = 3;
    duressAction = 0;
    easyReadFontForAll = 1;
    emptyOrNilPwPreferNilCheckFirst = 0;
    failedPinAttempts = 0;
    fileIdentifier = 0AAcD0j0uWrgDUk9PVA;
    fileName = "pwsafe.psafe3";
    forceOpenOffline = 0;
    hasAcknowledgedAppLockBiometricQuickLaunchCoalesceIssue = 0;
    hasBeenPromptedForConvenience = 1;
    hasBeenPromptedForQuickLaunch = 0;
    hasShownInitialOnboardingScreen = 1;
    hasUnresolvedConflicts = 0;
    hideIconInBrowse = 0;
    hideTotp = 0;
    hideTotpCustomFieldsInViewMode = 1;
    hideTotpInBrowse = 0;
    immediateSearchOnBrowse = 0;
    isEnrolledForConvenience = 1;
    isTouchIdEnabled = 1;
    keePassIconSet = 0;
    keyFileBookmark = "<Not Set>";
    lastSyncRemoteModDate = "568238064.382";
    likelyFormat = 0;
    lockEvenIfEditing = 0;
    longPressTapAction = 4;
    makeBackups = 1;
    maxBackupKeepCount = 10;
    nickName = "Dwight's";
    offlineDetectedBehaviour = 0;
    onboardingDoneHasBeenShown = 1;
    promptedForAutoFetchFavIcon = 0;
    quickTypeDisplayFormat = 0;
    quickTypeEnabled = 1;
    readOnly = 1;
    scheduleExportIntervalDays = 28;
    scheduledExport = 0;
    scheduledExportOnboardingDone = 0;
    showChildCountOnFolderInBrowse = 0;
    showConvenienceExpiryMessage = 1;
    showEmptyFieldsInDetailsView = 0;
    showExpiredInBrowse = 1;
    showExpiredInSearch = 1;
    showFlagsInBrowse = 1;
    showKeePass1BackupGroup = 0;
    showPasswordByDefaultOnEditScreen = 1;
    showQuickViewExpired = 0;
    showQuickViewFavourites = 1;
    showQuickViewNearlyExpired = 1;
    showRecycleBinInSearchResults = 0;
    storageProvider = 0;
    tapAction = 1;
    tripleTapAction = 8;
    tryDownloadFavIconForNewRecord = 1;
    uuid = "B2AE174D-0ABC-4F7D-B558-4DFF63D3264D";
}
[Ben-HP] on [OneDrive] Config
{
    auditConfig =     {
        auditInBackground = 1;
        caseInsensitiveMatchForDuplicates = 1;
        checkForCommonPasswords = 1;
        checkForDuplicatedPasswords = 1;
        checkForLowEntropy = 0;
        checkForMinimumLength = 0;
        checkForNoPasswords = 0;
        checkForSimilarPasswords = 0;
        checkHibp = 0;
        hibpCaveatAccepted = 0;
        hibpCheckForNewBreachesIntervalSeconds = 86400;
        lastKnownAuditIssueCount = 2;
        levenshteinSimilarityThreshold = "0.75";
        lowEntropyThreshold = 36;
        minimumLength = 12;
        showAuditPopupNotifications = 1;
        showCachedHibpHits = 0;
    };
    autoFillConvenienceAutoUnlockTimeout = 600;
    autoFillCopyTotp = 1;
    autoFillEnabled = 1;
    autoFillLastUnlockedAt = "645553161.7175111";
    autoFillOnboardingDone = 1;
    autoLockOnDeviceLock = 1;
    autoLockTimeoutSeconds = 120;
    browseItemSubtitleField = 1;
    browseSortField = 1;
    browseSortFoldersSeparately = 1;
    browseSortOrderDescending = 0;
    browseViewType = 0;
    colorizePasswords = 1;
    colorizeProtectedCustomFields = 0;
    conflictResolutionStrategy = 0;
    convenienceExpiryOnboardingDone = 1;
    convenienceExpiryPeriod = 336;
    couldNotConnectBehaviour = 0;
    detailsViewCollapsedSections =     (
        0,
        0,
        0,
        0,
        0,
        1
    );
    doNotShowRecycleBinInBrowse = 0;
    doubleTapAction = 4;
    duressAction = 0;
    easyReadFontForAll = 0;
    emptyOrNilPwPreferNilCheckFirst = 0;
    failedPinAttempts = 0;
    fileIdentifier = "{\"driveId\":\"b!FrXO2YRb-kGZKa4Ogj8lcW1vglHzb59Lrv0AU8tAinmrnXKJBPSbRYkQb_SZtzj_\",\"parentFolderId\":\"01UCUGGUF6Y2GOVW7725BZO354PWSELRRZ\"}";
    fileName = "Ben-HP.psafe3";
    forceOpenOffline = 0;
    hasAcknowledgedAppLockBiometricQuickLaunchCoalesceIssue = 0;
    hasBeenPromptedForConvenience = 1;
    hasBeenPromptedForQuickLaunch = 1;
    hasShownInitialOnboardingScreen = 1;
    hasUnresolvedConflicts = 0;
    hideIconInBrowse = 0;
    hideTotp = 0;
    hideTotpCustomFieldsInViewMode = 1;
    hideTotpInBrowse = 0;
    immediateSearchOnBrowse = 0;
    isEnrolledForConvenience = 1;
    isTouchIdEnabled = 1;
    keePassIconSet = 1;
    keyFileBookmark = "<Not Set>";
    lastSyncAttempt = "645553731.195011";
    lastSyncRemoteModDate = 645037152;
    likelyFormat = 0;
    lockEvenIfEditing = 0;
    longPressTapAction = 3;
    makeBackups = 1;
    maxBackupKeepCount = 10;
    nextScheduledExport = "647146767.459054";
    nickName = "Ben-HP";
    offlineDetectedBehaviour = 0;
    onboardingDoneHasBeenShown = 1;
    promptedForAutoFetchFavIcon = 0;
    quickTypeDisplayFormat = 0;
    quickTypeEnabled = 1;
    readOnly = 0;
    scheduleExportIntervalDays = 28;
    scheduledExport = 1;
    scheduledExportOnboardingDone = 1;
    showChildCountOnFolderInBrowse = 1;
    showConvenienceExpiryMessage = 1;
    showEmptyFieldsInDetailsView = 0;
    showExpiredInBrowse = 1;
    showExpiredInSearch = 1;
    showFlagsInBrowse = 1;
    showKeePass1BackupGroup = 0;
    showPasswordByDefaultOnEditScreen = 0;
    showQuickViewExpired = 0;
    showQuickViewFavourites = 1;
    showQuickViewNearlyExpired = 1;
    showRecycleBinInSearchResults = 0;
    storageProvider = 4;
    tapAction = 1;
    tripleTapAction = 8;
    tryDownloadFavIconForNewRecord = 1;
    uuid = "1081AE9A-7E68-42F6-9EA1-331E5F42766A";
}
[Copy of BenPasswords] on [Local] Config
{
    auditConfig =     {
        auditInBackground = 1;
        caseInsensitiveMatchForDuplicates = 1;
        checkForCommonPasswords = 1;
        checkForDuplicatedPasswords = 1;
        checkForLowEntropy = 0;
        checkForMinimumLength = 0;
        checkForNoPasswords = 0;
        checkForSimilarPasswords = 0;
        checkHibp = 0;
        hibpCaveatAccepted = 0;
        hibpCheckForNewBreachesIntervalSeconds = 86400;
        lastKnownAuditIssueCount = 3;
        levenshteinSimilarityThreshold = "0.75";
        lowEntropyThreshold = 36;
        minimumLength = 12;
        showAuditPopupNotifications = 1;
        showCachedHibpHits = 1;
    };
    autoFillConvenienceAutoUnlockTimeout = 180;
    autoFillCopyTotp = 1;
    autoFillEnabled = 1;
    autoFillLastUnlockedAt = "645458215.432474";
    autoFillOnboardingDone = 1;
    autoLockOnDeviceLock = 1;
    autoLockTimeoutSeconds = 60;
    browseItemSubtitleField = 1;
    browseSortField = 1;
    browseSortFoldersSeparately = 1;
    browseSortOrderDescending = 0;
    browseViewType = 0;
    colorizePasswords = 1;
    colorizeProtectedCustomFields = 0;
    conflictResolutionStrategy = 0;
    convenienceExpiryOnboardingDone = 0;
    convenienceExpiryPeriod = "-1";
    couldNotConnectBehaviour = 0;
    detailsViewCollapsedSections =     (
        0,
        0,
        0,
        0,
        1,
        1
    );
    doNotShowRecycleBinInBrowse = 0;
    doubleTapAction = 4;
    duressAction = 0;
    easyReadFontForAll = 0;
    emptyOrNilPwPreferNilCheckFirst = 0;
    failedPinAttempts = 0;
    fileIdentifier = "{\"filename\":\"Copy of BenPasswords.psafe3\",\"sharedStorage\":true}";
    fileName = "Copy of BenPasswords.psafe3";
    forceOpenOffline = 0;
    hasAcknowledgedAppLockBiometricQuickLaunchCoalesceIssue = 0;
    hasBeenPromptedForConvenience = 1;
    hasBeenPromptedForQuickLaunch = 0;
    hasShownInitialOnboardingScreen = 1;
    hasUnresolvedConflicts = 0;
    hideIconInBrowse = 0;
    hideTotp = 0;
    hideTotpCustomFieldsInViewMode = 1;
    hideTotpInBrowse = 0;
    immediateSearchOnBrowse = 0;
    isEnrolledForConvenience = 1;
    isTouchIdEnabled = 1;
    keePassIconSet = 1;
    keyFileBookmark = "<Not Set>";
    lastSyncAttempt = "645037078.021556";
    lastSyncRemoteModDate = "633907546.6803542";
    likelyFormat = 0;
    lockEvenIfEditing = 0;
    longPressTapAction = 3;
    makeBackups = 1;
    maxBackupKeepCount = 10;
    nickName = "Copy of BenPasswords";
    offlineDetectedBehaviour = 1;
    onboardingDoneHasBeenShown = 1;
    promptedForAutoFetchFavIcon = 0;
    quickTypeDisplayFormat = 0;
    quickTypeEnabled = 1;
    readOnly = 0;
    scheduleExportIntervalDays = 28;
    scheduledExport = 0;
    scheduledExportOnboardingDone = 0;
    showChildCountOnFolderInBrowse = 1;
    showConvenienceExpiryMessage = 1;
    showEmptyFieldsInDetailsView = 0;
    showExpiredInBrowse = 1;
    showExpiredInSearch = 1;
    showFlagsInBrowse = 1;
    showKeePass1BackupGroup = 0;
    showPasswordByDefaultOnEditScreen = 0;
    showQuickViewExpired = 0;
    showQuickViewFavourites = 1;
    showQuickViewNearlyExpired = 1;
    showRecycleBinInSearchResults = 0;
    storageProvider = 2;
    tapAction = 1;
    tripleTapAction = 8;
    tryDownloadFavIconForNewRecord = 1;
    uuid = "51FE0E2D-E9C2-4FBA-9749-304EFA4DFF0B";
}
--------------------
```

**Comment by strongbox-mark** [comment]
Hi Ben, this is a bug in the latest update, some people are seeing this. You still have pro, it's just a messaging issue. Fix on the way.

**Comment by strongbox-mark** [comment]
Fixed with 1.52.22. Apologies!
